nntrainer/graph/network_graph.cpp

```diff
@@ -249,19 +249,32 @@ int NetworkGraph::realizeMultiOutputType(
   return status;
 }
 
-/** TODO: this needs special attention */
+/**
+ * @fixme: the implementation assumes loss layer should always be at the last
+ * layer and the there is only one loss, this assumption is not true
+ */
```

Comment:
Is it true? we only support only on loss?? I thought we can run multi-output.

Reply:
Yes, for now, multi-output is supported only in the middle.